### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by lahiru823**
Fixed issues:
Policy group description show as undefined when a new group added
Throttling doesn't work for webapps
"Skip Creating Proxying WebApp" option not saved when creating a webapp
Change admin dashboard router log level

### CODE REVIEW

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag [hunk 1]
```diff
@@ -25,24 +25,24 @@
     var base = '/{context}/apis/{resource}';
 
     var API_LOCATIONS='/apis/v1/';
-    log.info('apis router hit');
+    log.debug('apis router hit');
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Surround the the debug block with an isDebugEnabled check.

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag [hunk 2]
```diff
@@ -25,24 +25,24 @@
...
     if((uriMatcher.match(pattern))|| (uriMatcher.match(base)) ){
         var options = uriMatcher.elements();
         var resource = options.resource;
 
         var apiDir = new File(API_LOCATIONS+resource+'.jag');
-        log.info('Looking for directory '+API_LOCATIONS+resource+'.jag');
+        log.debug('Looking for directory '+API_LOCATIONS+resource+'.jag');
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Surround the the debug block with an isDebugEnabled check.

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag [hunk 3]
```diff
@@ -25,24 +25,24 @@
...
         if(apiDir.isExists()){
-            log.info('Loading '+API_LOCATIONS+resource+'.jag');
+            log.debug('Loading '+API_LOCATIONS+resource+'.jag');
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Surround the the debug block with an isDebugEnabled check.

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag [hunk 4]
```diff
@@ -25,24 +25,24 @@
...
             include(API_LOCATIONS+resource+'.jag');
         }
         else{
-            log.info('Failed to locate resource');
+            log.debug('Failed to locate resource');
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Surround the the debug block with an isDebugEnabled check.

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag [hunk 5]
```diff
@@ -25,24 +25,24 @@
...
             response.sendError(404,'Resource does not exist');
         }
     }
     else{
-        log.info('Failed');
+        log.debug('Failed');
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Surround the the debug block with an isDebugEnabled check.

File: modules/jaggery-apps/app-admin-dashboard-web/site/themes/default/templates/xacmlpolicy-task/js/xacmlpolicy.js [hunk 6]
```diff
@@ -226,6 +224,18 @@ function savePolicyPartial() {
     var isSharedPartial = true;
     if (editedpolicyPartialId == 0) { //add
 
+        //check if the name is already saved
+        if (policyPartialsArray.length > 0) {
```
Comment (rushmin) [inline modules/jaggery-apps/app-admin-dashboard-web/site/themes/default/templates/xacmlpolicy-task/js/xacmlpolicy.js]: Length check it not needed.

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag (hunk 1 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Already fixed with a subsequent commit

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag (hunk 2 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Already fixed with a subsequent commit

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag (hunk 3 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Already fixed with a subsequent commit

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag (hunk 4 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Already fixed with a subsequent commit

File: modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag (hunk 5 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/controllers/apis-router.jag]: Already fixed with a subsequent commit

File: modules/jaggery-apps/app-admin-dashboard-web/site/themes/default/templates/xacmlpolicy-task/js/xacmlpolicy.js (hunk 6 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-admin-dashboard-web/site/themes/default/templates/xacmlpolicy-task/js/xacmlpolicy.js]: fixed with https://github.com/wso2/carbon-appmgt/commit/5d24df8ee1dd975bed2470e63e7c2193c87a5550

**Pull request merged**